fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -440,7 +441,9 @@ private boolean processDropColumn(DropColumnClause alterClause, OlapTable olapTa
                         newBfCols.add(bfCol);
                     }
                 }
-                olapTable.setBloomFilterInfo(newBfCols, olapTable.getBfFpp());
```

Comment:
why remove?

Reply:
Record the new bf columns to `propertyMap`, the actual bf columns drop operation is happening in the following schema change job which created by directly schema change.